cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,103 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
 
-  });
+import registratePageObject from '../support/pages/registrate.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import homePageObject from '../support/pages/home.pageObject';
 
-  beforeEach(() => {
+const registrate = new registratePageObject();
+const settings = new SettingsPageObject();
+const homePage = new homePageObject();
+const faker = require('faker');
 
-  });
+const userData = {
+  usernameData: faker.name.firstName().toLowerCase(),
+  userBio: 'Burning ass',
```

Comment:
Instead of hardcoding the userBio, you should generate it using faker. This would ensure the data is unique for every test run.